rust/src/quic/frames.rs

```diff
@@ -249,20 +251,54 @@ fn quic_get_tls_extensions(
                     dash = true;
                 }
                 ja3.push_str(&u16::from(etype).to_string());
+                if let Some(ref mut ja4) = ja4 {
+                    ja4.add_extension(etype)
+                }
                 let mut values = Vec::new();
                 match e {
+                    TlsExtension::SupportedVersions(x) => {
+                        let mut max: TlsVersion = TlsVersion::Ssl30;
+                        for version in x {
+                            if u16::from(*version) > u16::from(max) {
+                                max = *version;
```

Comment:
I do not see this max thing in app-layer-ssl.c
Why ?

Reply:
Will be handled differently now.